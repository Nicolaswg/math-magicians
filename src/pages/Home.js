import React from 'react';
import style from './Pages.module.css';

export default function Home() {
  return (
    <>
      <div className={style.homeContainer}>
        <h2 className={style.title}> Welcome to our page!</h2>
        <div>
          <p className={style.content}>
            Lorem ipsum, dolor sit amet consectetur
            adipisicing elit. Dolorum ut molestiae animi
            itaque esse nulla veritatis sed earum officiis
            Lorem ipsum, dolor sit amet consectetur
            animi itaque esse nulla veritatis sed
            adipisicing elit. Dolorum ut molestiae animi
            adipisicing elit. Dolorum ut molestiae animi
            adipisicing elit. Dolorum ut molestiae animi
            adipisicing elit. Dolorum ut molestiae animi
            earum officiis culpa! Mollitia
          </p>
        </div>
      </div>
    </>
  );
}
