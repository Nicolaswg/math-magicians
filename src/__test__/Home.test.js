import renderer from 'react-test-renderer';
import Home from '../pages/Home';

it('Render Home correctly', () => {
  const tree = renderer.create(
    <Home />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
