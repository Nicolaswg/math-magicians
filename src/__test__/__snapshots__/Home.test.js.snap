// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Home correctly 1`] = `
<div
  className="homeContainer"
>
  <h2
    className="title"
  >
     Welcome to our page!
  </h2>
  <div>
    <p
      className="content"
    >
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Dolorum ut molestiae animi itaque esse nulla veritatis sed earum officiis Lorem ipsum, dolor sit amet consectetur animi itaque esse nulla veritatis sed adipisicing elit. Dolorum ut molestiae animi adipisicing elit. Dolorum ut molestiae animi adipisicing elit. Dolorum ut molestiae animi adipisicing elit. Dolorum ut molestiae animi earum officiis culpa! Mollitia
    </p>
  </div>
</div>
`;
